README.md

```diff
@@ -1726,6 +1722,41 @@ To retrieve values using the key, use `map.get()`.
 
 As each key is unique, putting foo as -23 overwrites the previous value of 42 so when you get foo, the value will be -23.
 
+## JavaScript Object Notation (JSON)
+
+JavaScript Object Notation (JSON) is a way of storing data and exchanging data between a browser and a server. Although JSON uses the JavaScript syntax, its format is text so it can be read and used as a data format by any programming language.
+
+JSON has the following data types:
+
+- a string
+- a number
+- an object (JSON object)
+- an array
+- a boolean
+- null
+
+#### Example of a JSON Object containing an array of objects:
+``` json
+{ "Dogs":
```

Comment:
Just a formatting suggestion (That's a bit more standard)

- `Dogs` -> `dogs`
- two spaces for indentation consistently
- beginning of value directly after colon

```json
{
  "dogs": [
    {
      "name": "Bertie",
      "age": 5
    },
    {
      "name": "Pia",
      "age": 2
    }
  ]
}
```